packages/pagination/src/elements/Pagination.tsx

```diff
@@ -35,18 +31,27 @@ export interface IPaginationProps extends Omit<HTMLAttributes<HTMLUListElement>,
    */
   totalPages: number;
   /**
-   * The number of pages to pad the currentPage with
-   * when determining the Gap placement
+   * The number of pages to pad the current page with when page gaps are
+   * displayed
    */
   pagePadding?: number;
   /**
-   * @param {Any} currentPage - The newly selected page
+   * Position for the leading and trailing gap indicator, if needed based on
+   * current and total pages
    */
-  onChange?: (updatedCurrentPage: number) => void;
+  pageGap?: number;
   /**
-   * Allows custom props to be applied to each page element. Useful for QA attributes and localization.
+   * @param {Any} currentPage - The currently selected page
```

Comment:
These JSDoc values are already created by TSDoc in the prop-sheet. Do we need to duplicate these here?

Reply:
Without the `param`, the argument isn't documented like I want it to be.

![Screen Shot 2020-01-27 at 3 20 20 PM](https://user-images.githubusercontent.com/143773/73222635-9438ea80-4118-11ea-9432-094175ecd1d5.png)